lib/shared/log/tasks.js

```diff
@@ -12,7 +15,7 @@ function logTasks(tree, opts, getTask) {
   var maxDepth = opts.tasksDepth;
   if (typeof maxDepth !== 'number') {
     maxDepth = 50;
-  } else if (maxDepth < 1) {
+  } else /* istanbul ignore if */ if (maxDepth < 1) {
```

Comment:
Why add this comment now?

Reply:
I'll remove this comment and add a test case to pass it.